face_track/face_track.py

```diff
@@ -144,8 +147,20 @@ def __init__(self):
 
 		# Which face to look at
 		# rospy.Subscriber(self.TOPIC_FACE_TARGET, xxxFaceEvent, xxxself.face_event_cb)
-		rospy.Subscriber("perceived_text", String,
-			self.user_said_cb)
+		#rospy.Subscriber("perceived_text", String,
+		#	self.user_said_cb)
+		self.speech_start_at=datetime.datetime.now()
+		self.speech_end_at=datetime.datetime.now()
+		self.speech_elapse_ms=0.0
+		self.speech_end_elapse_ms=0.0;
+		rospy.Subscriber("chatbot_speech", ChatMessage,
+			self.stt_cb)
+
+		# rospy.Subscriber("chatbot_speech", ChatMessage,
+		# 	self.user_said_cb)
+		# rospy.Subscriber("chat_events", String,
+		# 	self.speech_event_cb)
```

Comment:
please do not submit code that is commented out.  Either the code works, or it doesn't.  Comment lines should be for comments, and not for code.